bundles/org.openhab.binding.mybmw/README.md

```diff
@@ -132,9 +132,10 @@ They differ for each vehicle type, build-in sensors and activated services.
 
 | Channel Group ID                 | Description                                       | conv | phev | bev_rex | bev |
 |----------------------------------|---------------------------------------------------|------|------|---------|-----|
+| [update](#vehicle-update)        | Overall vehicle status                            |  X   |  X   |    X    |  X  |
 | [status](#vehicle-status)        | Overall vehicle status                            |  X   |  X   |    X    |  X  |
-| [range](#range-data)             | Provides mileage, range and charge / fuel levels  |  X   |  X   |    X    |  X  |
 | [doors](#doors-details)          | Detials of all doors and windows                  |  X   |  X   |    X    |  X  |
```

Comment:
| [doors](#doors-details)          | Details of all doors and windows                  |  X   |  X   |    X    |  X  |